src/main/scala/com/crealytics/spark/excel/WorkbookReader.scala

```diff
@@ -26,7 +26,8 @@ import org.apache.poi.hssf.usermodel.HSSFWorkbookFactory
 import org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource
 import org.apache.poi.util.IOUtils
 import org.apache.poi.xssf.usermodel.XSSFWorkbookFactory
-import scala.collection.JavaConverters.mapAsScalaMapConverter
+
+import scala.jdk.CollectionConverters._
```

Comment:
Just a heads-up: In my current project we just ran into a problem where including `scala-collection-compat` into the fat jar yielded errors because the included `scala` package (from `scala.jdk`) prevented the actual Scala package from being loaded. We are not bundling `scala-collection-compat` therefore I hope this won't be an error, but in case someone complains this would be the place to look at.

Reply:
revert that change, did that by accident